src/main/asciidoc/jdbc.adoc

```diff
@@ -86,108 +84,115 @@ class ApplicationConfig {
+If the identifier property is `null`, then the entity is assumed to be new. Otherwise, it is assumed to not be new.
+|Implementing `Persistable`|If an entity implements `Persistable`, Spring Data JDBC delegates the new detection to the `isNew(…)` method of the entity.
 See the link:$$http://docs.spring.io/spring-data/data-commons/docs/current/api/index.html?org/springframework/data/domain/Persistable.html$$[JavaDoc] for details.
-|Implementing `EntityInformation`|You can customize the `EntityInformation` abstraction used in the `SimpleJdbcRepository` implementation by creating a subclass of `JdbcRepositoryFactory` and overriding the `getEntityInformation(…)` method accordingly.
+|Implementing `EntityInformation`|You can customize the `EntityInformation` abstraction used in the `SimpleJdbcRepository` implementation by creating a subclass of `JdbcRepositoryFactory` and overriding the `getEntityInformation(…)` method.
 You then have to register the custom implementation of `JdbcRepositoryFactory` as a Spring bean.
-Note that this should be rarely necessary. See the link:$$http://docs.spring.io/spring-data/data-jdbc/docs/current/api/index.html?org/springframework/data/jdbc/repository/support/JdbcRepositoryFactory.html$$[JavaDoc] for details.
+Note that this should rarely be necessary. See the link:$$http://docs.spring.io/spring-data/data-jdbc/docs/current/api/index.html?org/springframework/data/jdbc/repository/support/JdbcRepositoryFactory.html$$[JavaDoc] for details.
 |===============
 
 [[jdbc.entity-persistence.id-generation]]
-=== Id generation
+=== ID Generation
 
-Spring Data JDBC uses the id to identify entities.
-The id of an entity must be annotated with Spring Data's https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/annotation/Id.html[`@Id`] annotation.
+Spring Data JDBC uses the ID to identify entities.
+The ID of an entity must be annotated with Spring Data's https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/annotation/Id.html[`@Id`] annotation.
 
-When your data base has some autoincrement column for the id column the generated value will get set in the entity after inserting it into the database.
+When your data base has an auto-increment column for the ID column, the generated value gets set in the entity after inserting it into the database.
 
-One important constraint is that after saving an entity the entity must not be _new_ anymore.
-With autoincrement columns this happens automatically since the the id gets set by Spring Data with the value from the id column.
-If you are not using autoincrement columns you can use that using a `BeforeSave`-listener which sets the id of the entity (see below).
+One important constraint is that, after saving an entity, the entity must not be new any more.
+With auto-increment columns, this happens automatically, because the ID gets set by Spring Data with the value from the `id` column.
```

Comment:
Not sure if `id` is the correct formatting, I think it should be "ID column" as two lines above, since it is the identifying column, not necessarily the column named id.

Reply:
OK. I changed it.